src/Compilers/CSharp/Portable/Symbols/LocalSymbol.cs

```diff
@@ -25,6 +25,11 @@ protected LocalSymbol()
             get;
         }
 
+        /// <summary>
+        /// Syntax node that is used as the scope designator. Otherwise, null.
+        /// </summary>
+        internal abstract SyntaxNode ScopeDesignatorOpt { get; }
```

Comment:
I'd not require that in this change. Especially since we might not want to touch SynthesizedLocalKind this close to RTW. It might make sense to add a tracking issue on revisiting this.

---
In reply to: [96769556](https://github.com/dotnet/roslyn/pull/16593#discussion_r96769556) [](ancestors = 96769556,96762026)